ODLA/platforms/odla_popart/popart_config.cc

```diff
@@ -200,6 +213,7 @@ bool PopartConfig::get_pipeline_setting(const std::string& node_name,
     pipeline_stage = default_setting_iter->second[1];
     return true;
   }
+
   throw std::runtime_error(
```

Comment:
Why we kept this throw?

Reply:
exception is caught outside. in builder::createFromOnnx